server/managers/AudioMetadataManager.js

```diff
@@ -32,9 +32,11 @@ class AudioMetadataMangaer {
     var metadataFilePath = Path.join(outputDir, 'metadata.txt')
     await writeMetadataFile(libraryItem, metadataFilePath)
 
+    var coverPath = libraryItem.media.coverPath.replace(/\\/g, '/')
```

Comment:
`libraryItem.media.coverPath` could be null here, we need to check if the cover exists